src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+import { RegistrationCards } from "./Registration.config";
+import styles from "./Registration.module.sass";
+import Link from "next/link";
+
+const Registration = () => {
+  return (
+    <>
+      <div className={styles.reg_header} key="reg-header">
+        Logo
+      </div>
+      <div className={styles.reg_body} key="reg-body">
+        <span class="material-symbols-outlined">
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0"
+          />
+          arrow_back_ios
+        </span>
+        <Link href="/">Atpakaļ</Link>
+        <div className={styles.reg_content}>
+          <h1>Pievienojies kā ēnotājs vai kā profesionālis vai uzņēmums?</h1>
+          <h3>
+            Autem sunt ratione asperiores odit nobis consequuntur quo sequi.
+          </h3>
+          <div className={styles.reg_cards} key="reg-cards">
+            {RegistrationCards.map((item) => (
+              <div className={styles.card} key={item.link}>
```

Comment:
I would suggest to move this to own component **RegistrationCard**, and pass required arguments